ui/app/components/send/currency-display.js

```diff
@@ -82,16 +82,21 @@ CurrencyDisplay.prototype.getConvertedValueToRender = function (nonFormattedValu
     numberOfDecimals: 2,
     conversionRate,
   })
-  convertedValue = Number(convertedValue).toFixed(2)
 
-  const upperCaseCurrencyCode = convertedCurrency.toUpperCase()
-
-  return currencies.find(currency => currency.code === upperCaseCurrencyCode)
-    ? currencyFormatter.format(Number(convertedValue), {
-      code: upperCaseCurrencyCode,
-    })
-    : convertedValue
-}
+  if (conversionRate == 0 && nonFormattedValue != 0) {
```

Comment:
A few comments on this:

- I think `conversionRate == 0` is a pretty good check here. It will match `0`, `'0'`, `'0.00'`, `''`. I wonder if we should also check whether conversion rate is `null` or `undefined`?
- Do we only want to return null if `nonFormattedValue != 0`?
- instead of setting `convertedValue` to `null` and the returning the variable, just `return null` would also work